pyscf/pbc/gto/test/test_rcut.py

```diff
@@ -54,6 +54,21 @@ def test_rcut(self):
             self.assertTrue(abs(t1-t0).max() < prec*1e-0)
             self.assertTrue(abs(s1-s0).max() < prec*1e-1)
 
+    def test_loose_rcut(self):
+        kpts = cell.make_kpts([2,2,2])
+        t0 = numpy.asarray(cell.pbc_intor('int1e_kin_sph', hermi=1, kpts=kpts))
+        s0 = numpy.asarray(cell.pbc_intor('int1e_ovlp_sph', hermi=1, kpts=kpts))
+        cell1 = cell.copy()
+        cell1.use_loose_rcut = True
+        for i in range(3):
+            cell1.precision = prec = 1e-8 * 10**i
+            cell1.build()
+            t1 = numpy.asarray(cell1.pbc_intor('int1e_kin_sph', hermi=1, kpts=kpts))
+            s1 = numpy.asarray(cell1.pbc_intor('int1e_ovlp_sph', hermi=1, kpts=kpts))
+            print(prec, 'error = ', abs(t1-t0).max(), abs(s1-s0).max())
+            self.assertTrue(abs(t1-t0).max() < prec*2e3)
```

Comment:
Is an error ~1e3 too large? Should the loose_rcut be tested using a different metric like the value of orbital or electron density at that rcut?

Reply:
I relace the test by checking the AO value at `cell.rcut` smaller than `cell.precision`.